### ISSUE TITLE
++ does not work in REPL

### ISSUE THREAD

**Issue by Ivo-Balbaert** [vlang/v#3333]
**V version:** V 0.1.24 a6a9958
**OS:** Windows 10 and Linux

**What did you do?**
```
C:\Users\CVO>v
V 0.1.24 a6a9958
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
1
>>> i
1
>>> println(i)
1
```

**What did you expect to see?**
```
C:\Users\CVO>v
V 0.1.24 a6a9958
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
2
>>> i
2
>>> println(i)
2
```
 
**What did you see instead?**
see **What did you do**

**Comment by eastern-shivanjan** [comment]
Feels good to back on V.
As per your expected output
i.e.
```
C:\Users\CVO>v
V 0.1.24 a6a9958
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
2
>>> i
2
>>> println(i)
2
```

I think the output should be instead since you don't want to print out the updated value every time you increment it right?
```
C:\Users\CVO>v
V 0.1.24 a6a9958
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
>>> i
2
>>> println(i)
2
```

I have a fix for this and I'm going to push it in a few minutes

**Comment by Glitchfix** [comment]
Feels good to back on V.
As per your expected output
i.e.

C:\Users\CVO>v
V 0.1.24 a6a9958
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
2
>>> i
2
>>> println(i)
2
I think the output should be instead since you don't want to print out the updated value every time you increment it right?

C:\Users\CVO>v
V 0.1.24 a6a9958
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
>>> i
2
>>> println(i)
2
I have a fix for this and I'm going to push it in a few minutes

**Comment by Ivo-Balbaert** [comment]
Thanks! I tested with the actual version 3425934, and I can confirm that the problem is solved on Linux Ubuntu. However the problem remains on Windows 10, here is a REPL printout:

C:\Users\CVO>v
V 0.1.24 3425934
Use Ctrl-C or `exit` to exit
>>> mut i := 1
1
>>> i
1
>>> i++
1
>>> i
1
>>> println(i)
1
>>> i++
1
>>> println(i)
1
>>>

**Comment by alexesprit** [comment]
@Ivo-Balbaert that's odd, I'm on Windows 10, and it's fixed for me (I tested on V 0.1.24 3425934 and c1cc203). Interesting, but according to your logs, you still have the previous behavior, where `i` value is printed after assignment and increment (in that fixed version it should not be printed):
```
C:\Users\CVO>v
V 0.1.24 3425934
Use Ctrl-C or `exit` to exit
>>> mut i := 1
1 <- should not be printed
>>> i
1
>>> i++
1 <- should not be printed
>>> i
1
>>> println(i)
1
>>> i++
1 <- should not be printed
>>> println(i)
1
>>>
```

**Comment by Ivo-Balbaert** [comment]
I always did v up, but something must have been left over from the previous install. I now dit a completely new install with git clone and make, and now it works like it should be:

c:\v>v
V 0.1.24 760448a
Use Ctrl-C or `exit` to exit
>>> mut i := 1
>>> i
1
>>> i++
>>> i
2
>>> i++
>>> println(i)
3
>>>

Thanks and closing!
